crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -16,7 +16,9 @@ pub mod graph {
     #[derive(Debug, Hash, PartialEq, Eq, Clone, RenderLabel)]
     pub enum Node3d {
         MsaaWriteback,
-        Prepass,
+        EarlyPrepass,
```

Comment:
Is the early prepass running the _full_ prepass? E.g. for deferred is it doing the gbuffer rendering too?

Not sure, but it might make more sense to do depth only in the early pass, and then depth + other attachments in the late pass.

Reply:
I'm pretty sure that Griffin said that you usually split the full prepass into early and late phases rather than having a separate z-prepass, but I'd rather not make that change here as this patch is too big as it is. Instead I just documented that occlusion culling is currently incompatible with deferred. We can add support for deferred in a followup.